### PR TITLE
Fix/DEV-228

### PR DESCRIPTION
**Pull request by armine-fliplet**
### Product areas affected

Fliiplet Form Builder

### What does this PR do?

Ensures the Map field is displayed in the form settings or on the screen if it is marked as required and user is able to change and save settings like "Field name", "Field label", "Description" etc for any form field.

### JIRA ticket

[DEV-227](https://weboo.atlassian.net/browse/DEV-227)
[DEV-228](https://weboo.atlassian.net/browse/DEV-228)

[DEV-227]: https://weboo.atlassian.net/browse/DEV-227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEV-228]: https://weboo.atlassian.net/browse/DEV-228?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Arpanexe — APPROVED** [review]
LGTM

File: js/components/map.js [hunk 1]
```diff
@@ -239,14 +239,13 @@ Fliplet.FormBuilder.field('map', {
...
   validations: function() {
-    var $vm = this;
     var rules = {
       value: {}
     };
 
     if (this.required && !this.readonly) {
       rules.value.required = function() {
-        return $vm.value.address;
+        return  this.value && this.value.address ? true : false;
```
Comment (dwfliplet) [inline js/components/map.js:248]:
```suggestion
        return  !!(this.value?.address);
```

Comment (armine-fliplet) [reply js/components/map.js]: @dwfliplet I got ESLint error for ?

Comment (dwfliplet) [reply js/components/map.js]: Then go for:
```
 return  !!(this.value && this.value.address);
```

**Review by dwfliplet — APPROVED** [review]

**Pull request merged**